src/__test__/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapGroupBySettings.test.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
```

Comment:
Martin, do you develop with the UI open as a workspace in VsCode, so that the code is auto-formatted and corrected after save? I just checked out your changes locally and this file looks full of linting errors (commas, formatting, etc.)